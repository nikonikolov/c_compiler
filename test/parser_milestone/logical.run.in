# 2 "logical.run.in" 3 4
int x()
{}


int g, b, d, f=a;


int zz(int a, int b, int c)
{


	while(a+a==a){
		a=b;
	}

}