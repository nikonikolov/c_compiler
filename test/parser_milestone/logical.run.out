FUNCTION : bla
    PARAM : t
    PARAM : p
    PARAM : c
SCOPE
    (b)!=(((((a)*(b))-((((c)/(g))*(g))%(h)))-(b))+((a)|(b)))
    (b)==(((((a)*(b))-((((c)/(g))*(g))%(h)))-(b))+((a)|(b)))
    (a)||(b)
    (a)&&(b)
    (a)||(b)
    (a)=(!(b))
    (a)=(~(b))
    (A)||((b)&&((c)+(d)))
    (A)||((b)&&(c))
    ((a)|(b))||(((a)&(b))&&(c))
